src/iris/api.py

```diff
@@ -1267,11 +1302,54 @@ def on_get(self, req, resp):
              ]
         You can also search for plans that have specific targets in their steps by using the field 'target'
 
+        *Allowed filter parameters:*
+
+        - id
+        - name
+        - tracking_type
+        - tracking_key
+        - created
+        - creator
+        - active
+        - tags
+
+        for querying tags a "tag_" prefix should be added before the tag name and then it can be used just as any other normal field
+        e.g. if we want to query for a plan that has a tag "service" with a value "foo" we would query: /v0/plan?tag_service=foo
+
         **example request**
 
         GET /v0/plans?target=foo&active=1 HTTP/1.1
 
+        Additionally adding the parameter "counts" will return a dictionary with unique values of each field and their counts as well as a total count
+
+        **Example request**:
+
+        .. sourcecode:: http
+
+            GET /v0/plans?counts=true&fields=creator  HTTP/1.1
+            Host: example.com
+
+        **Example response**:
+
+        .. sourcecode:: http
+
+            HTTP/1.1 200 OK
+            Content-Type: application/json
+
+            {
+                "field_counts": {
+                    "creator": {
+                        "user_foo": 10,
+                        "user_bar": 20,
+                        "user_baz": 5
+                    }
+                },
+                "total_count": 35
+            }
+
         '''
+        counts_only = req.get_param('counts')
```

Comment:
Could this be get_param_as_bool() instead?

Reply:
changed